src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -69,14 +73,67 @@ def make_artifacts(
     "-b",
     "--mark-best",
     is_flag=True,
+    help="Marks this version of results with a 'best' symlink.",
 )
 @click.option(
     "-t",
     "--test-run",
     is_flag=True,
+    help="Skips updating the 'latest' symlink with this version of results.",
 )
 @click.option(
-    "-a", "--artifact_path", type=click.Path(exists=True), default=paths.DEFAULT_ARTIFACT
+    "-a",
```

Comment:
Cosmetic nit: We've been using `-a` for the artifact option since we added it to PRL make_results, but it's been bugging me since we use `-i` for the artifact in all the other commands.